inc/containerdisplaycondition.class.php

```diff
@@ -77,7 +77,9 @@ public static function installBaseData(Migration $migration, $version)
                   PRIMARY KEY                         (`id`),
                   KEY `plugin_fields_containers_id_itemtype`       (`plugin_fields_containers_id`, `itemtype`)
                ) ENGINE=InnoDB DEFAULT CHARSET={$default_charset} COLLATE={$default_collation} ROW_FORMAT=DYNAMIC;";
-            $DB->doQuery($query) or die($DB->error());
+            if (!$DB->doQuery($query)) {
+                throw new \RuntimeException('Error creating plugin_fields_containers_displayconditions table: ' . $DB->error());
+            }
```

Comment:
Not necessary, because doQuery() already throws an exception.
```suggestion
            $DB->doQuery($query);
```

_I'm not commenting on all cases, but this applies to all of them._